test/test.basic.js

```diff
@@ -148,7 +148,7 @@ describe("Basics", function () {
             var positive_count = 0;
```

Comment:
good catch, I would also change line 147 to better describe the test, since now we are technically testing 'it is always a positive number or 0'